crates/sui-core/src/consensus_adapter.rs

```diff
@@ -334,22 +512,6 @@ impl ConsensusAdapter {
             epoch_store.record_epoch_pending_certs_process_time_metric();
         }
 
-        let processed_waiter = epoch_store
-            .consensus_message_processed_notify(transaction.key())
-            .boxed();
-        let (await_submit, position) =
-            Self::await_submit_delay(epoch_store.committee(), &self.authority, &transaction);
-        let mut guard = InflightDropGuard::acquire(&self);
```

Comment:
I've left a comment bellow on another part of the code, but this is important to stay here as it measures critical end to end latencies which we won't be to acquire otherwise